code/game/objects/items/tools/wrench.dm

```diff
@@ -103,9 +103,9 @@
 		on = TRUE
 		force = 6
 		w_class = WEIGHT_CLASS_NORMAL
-		throwforce = 8
+		throwforce = 10
 		tool_behaviour = TOOL_WRENCH
-		toolspeed = 1
+		toolspeed = 0.5

```

Comment:
undocumented change, please put this in the PR body

also consider refactoring this to use ternary operators like my above suggestion